packages/@aws-cdk/aws-s3-deployment/README.md

```diff
@@ -177,6 +178,7 @@ new s3deploy.BucketDeployment(this, 'DeployWebsite', {
   storageClass: StorageClass.INTELLIGENT_TIERING,
   serverSideEncryption: ServerSideEncryption.AES_256,
   cacheControl: [CacheControl.setPublic(), CacheControl.maxAge(cdk.Duration.hours(1))],
+  accessControl: s3.BucketAccessControl.BUCKET_OWNER_FULL_CONTROL,
```

Comment:
We usually like to add a couple of sentences on what this feature, when it should be used and any caveats.

Some require a separate section but most just a couple of sentences.

Reply:
So `accessControl` is one of system metadata properties like `serverSideEncryption` and etc. It seems like there is no additional information provided for other system metadata properties here. I have added a note about how system metadata maps to `aws s3 sync` arguments and where additional information can be obtained.